provider/ovh/ovh.go

```diff
@@ -146,12 +146,12 @@ func (p *OVHProvider) ApplyChanges(ctx context.Context, changes *plan.Changes) e
 
 	allChanges := make([]ovhChange, 0, countTargets(changes.Create, changes.UpdateNew, changes.UpdateOld, changes.Delete))
 
-	allChanges = append(allChanges, newOvhChange(ovhCreate, changes.Create, zones, records)...)
-	allChanges = append(allChanges, newOvhChange(ovhCreate, changes.UpdateNew, zones, records)...)
-
 	allChanges = append(allChanges, newOvhChange(ovhDelete, changes.UpdateOld, zones, records)...)
 	allChanges = append(allChanges, newOvhChange(ovhDelete, changes.Delete, zones, records)...)
 
+	allChanges = append(allChanges, newOvhChange(ovhCreate, changes.UpdateNew, zones, records)...)
+	allChanges = append(allChanges, newOvhChange(ovhCreate, changes.Create, zones, records)...)
+
 	log.Infof("OVH: %d changes will be done", len(allChanges))
```

Comment:
Why is there an order change here ?

Reply:
My understanding for the OVH provider, which we use, is that it doesn't actually update records, but POST on
`/records` the new entries 

Putting the delete actions (update old and delete) before would ensure the records are clean when doing a migration or a new creation 

I'll revert this change on my side and test again with the latest changes on TXT to confirm the behavior or not

Reply:
Yeah it doesn't seem incidental at all on both config so I was mistaken... 
I'll revert this to avoid useless changes